infra/cifuzz.py

```diff
@@ -0,0 +1,77 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Module used by CI tools in order to interact with fuzzers.
+This module helps CI tools do the following
+  1. build fuzzers
+  2. run fuzzers
+Eventually it will be used to help CI tools determine which fuzzers to run.
+"""
+
+import argparse
+import os
+import tempfile
+
+import build_specified_commit
+import repo_manager
+import utils
+
+
+def main():
+  """Connects Fuzzers with CI tools."""
+  parser = argparse.ArgumentParser(
+      description='Help CI tools manage specific fuzzers.')
+
+  subparsers = parser.add_subparsers(dest='command')
+  build_fuzzer_parser = subparsers.add_parser(
+      'build_fuzzers', help='Build an OSS-Fuzz projects fuzzers.')
```

Comment:
nit: would it make sense to have shorter commands, just `build` and `run`?

Reply:
Acknowledged. As I mentioned before I think this could get confusing with Actions having a build project function.